matrix-generator.js

```diff
@@ -1,5 +1,22 @@
+"use strict";
+
 function matrixGenerator(rows, columns) {
   // do work here
+
+  let final = [];
+
+  for (x = 0; x < rows; x++) {
+    let columnArr = [];
+
+    for (y = 0; y < columns; y++) {
+      // needs to be random num 1 - 100
+      columnArr.push(Math.floor(Math.random * 101));
+      // sort array
+      columnArr.sort();
```

Comment:
sort needs to take in a function that describes how the array should be sorted: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort